src/assets/localization.scss

```diff
@@ -0,0 +1,7 @@
+@import url('http://fonts.cdnfonts.com/css/iranian-sans');
```

Comment:
I believe it will import for all languages. It should be imported only if lang=fa.

If there is no way to do that, using system font is perferred.

Reply:
Yes @louislam, you are right. So I have deleted the import and enhanced the font-family to use system fonts.